inc/cart.php

```diff
@@ -88,4 +88,4 @@
 
 $selectStmt->close();
 $conn->close();
-?>
\ No newline at end of file
+?>
```

Comment:
To stejné co jsem psal již jinde. Ve zkratce:
1. Nepřehledný kód.
2. Oddělení logiky a prezentace.
3. 3x pěkně pojmenované funkce, aby ten co čte věděl, co tím chtěl autor říct.
4. Neošetřené vstupy.